package.json

```diff
@@ -16,16 +16,21 @@
   },
   "homepage": "https://skia.org",
   "esy": {
+    "buildEnv": {
+      "PATH": "#{self.target_dir : $PATH}"
+    },
     "buildsInSource": true,
     "build": [
+      "cp /usr/bin/python ./python.exe",
       [
         "python",
         "tools/git-sync-deps"
       ],
       [
         "bin/gn",
         "gen",
-        "#{self.target_dir}/out/Debug"
+        "#{self.target_dir}/out/Debug",
```

Comment:
Really nitpicking here, but maybe we want to call the directory `out/Release` now that it's not a debug build anymore. Just so that nobody gets confused here :) Also aligned with the `cp` command in `install`, then.